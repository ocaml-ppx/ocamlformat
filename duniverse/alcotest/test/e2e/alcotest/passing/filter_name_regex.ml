let () =
  let open Alcotest in
  let id () = () in
  run
    ~argv:[| ""; "test"; "basic-.*-(a|c)" |]
    __FILE__
    [
      ( "basic-run-a",
        [ test_case "Executed" `Quick id; test_case "Also executed" `Quick id ]
      );
      ("basic-run-b", [ test_case "Skipped" `Quick id ]);
      ("basic-run-c", [ test_case "Executed" `Quick id ]);
      ("complex-run-a", [ test_case "Skipped" `Quick id ]);
    ]
