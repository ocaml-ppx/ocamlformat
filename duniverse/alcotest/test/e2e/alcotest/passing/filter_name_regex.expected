Testing `test/e2e/alcotest/passing/filter_name_regex.ml'.
This run has ID `<uuid>'.

  [OK]          basic-run-a            0   Executed.
  [OK]          basic-run-a            1   Also executed.
  [SKIP]        basic-run-b            0   Skipped.
  [OK]          basic-run-c            0   Executed.
  [SKIP]        complex-run-a          0   Skipped.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 3 tests run.
